.github/workflows/stale_issues.yml

```diff
@@ -0,0 +1,45 @@
+name: "Close stale issues"
+
+# Controls when the action will run.
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    name: Stale issue job
+    steps:
+    - uses: aws-actions/stale-issue-cleanup@v2
+      with:
+        # Setting messages to an empty string will cause the automation to skip
+        # that category
+        ancient-issue-message: We have noticed this issue has not recieved attention in 3 years. We will close this issue for now. If you think this is in error, please feel free to comment and reopen the issue.
```

Comment:
Should issues necessarily close if they are feature requests, even if they are 3 years old? I can see value keeping them open as tracking and a gauge of community feedback, even if they have fewer than 10 upvotes.

Reply:
We can use the `no-autoclose` label on issues which we want the action to ignore.